src/pages/CampusAlumniEndowmentStudioPage.tsx

```
@@ -0,0 +1,218 @@
+import React, { useState, useEffect } from 'react';
+import { CampusAlumniEndowmentEngine } from '../../backend/src/services/CampusAlumniEndowmentEngine';
```

ESLint

[error] Replace `'../../backend/src/services/CampusAlumniEndowmentEngine'` with `"../../backend/src/services/CampusAlumniEndowmentEngine"`

(prettier/prettier)